docs/websockets.md

```diff
@@ -2,6 +2,10 @@
 Starlette includes a `WebSocket` class that fulfils a similar role
 to the HTTP request, but that allows sending and receiving data on a websocket.
 
+**Note:** Starlette doesn't implement the websocket protocol, this must be handled
+by your ASGI web server. For example, uvicorn will need either the `wsproto` or
+`websockets` PyPI package installed.
```

Comment:
I don't think the sentence is very clear. 🤔 

Starlette does implement the WebSocket protocol.

Reply:
How would you describe the subsystem that Starlette requires uvicorn to provide? That main thing is that I was running uvicorn without websockets installed and it had me very confused! I had assumed that all the websocket support was built into Starlette, but there was a piece missing.

Reply:
Perhaps this is better?

> For websockets to work, the ASGI web server that calls Starlette will also need support for websockets. For example, uvicorn will need either the `wsproto` or `websockets` PyPI package installed.